staging/src/k8s.io/client-go/tools/cache/controller_test.go

```diff
@@ -250,6 +250,7 @@ func TestHammerController(t *testing.T) {
 	wg := sync.WaitGroup{}
 	const threads = 3
 	wg.Add(threads)
+	var totalOps uint64
```

Comment:
IIUIC this will be equal to threads * 100, so we should not need to count, right?